src/api/app/models/history_element/review_accepted.rb

```diff
@@ -5,7 +5,7 @@ def description
     end
 
     def user_action
-      'accepted review'
+      'accepted the request'
```

Comment:
It just didn't felt right saying _accepted review_ because _one **gives** a review, and the result of the review can be an accepted or declined request_ . So one either accepts or declines a request by submitting a review. A review should not have a declined or accepted state actually
The purpose of this text fix was to shortcut a bigger refactoring in the logic and the relation between these two objects.